detectron2/structures/boxes.py

```diff
@@ -211,8 +211,7 @@ def nonempty(self, threshold: float = 0.0) -> torch.Tensor:
         keep = (widths > threshold) & (heights > threshold)
         return keep
 
-    @torch.jit.unused
-    def __getitem__(self, item: Union[int, slice, torch.BoolTensor]):
+    def __getitem__(self, item):
```

Comment:
mention the type as comment

Reply:
it seems that the type is already mentioned in comment, do you mean make it more detailed?

Reply:
didn't notice that! this is fine